src/compact.jl

```diff
@@ -112,14 +121,15 @@ macro compact(fex, _kwexs...)
   # make strings
   layer = "@compact"
   setup = NamedTuple(map(ex -> Symbol(string(ex.args[1])) => string(ex.args[2]), kwexs))
-  input = join(fex.args[1].args, ", ")
+  input = try join(fex.args[1].args, ", ") catch e 
+    @warn "function stringifying does not yet handle all cases, using empty string"
+    ""
+  end
```

Comment:
```suggestion
  input =
      try
          fex_args = fex.args[1]
          isa(fex_args, Symbol) ? string(fex_args) : join(fex_args.args, ", ")
      catch e 
        @warn "function stringifying does not yet handle all cases, using empty string"
        ""
      end
```

Reply:
(Required to get one-argument functions like `@compact(x -> x * a; a=rand())` working)